subsys/bluetooth/host/conn_internal.h

```diff
@@ -182,12 +182,20 @@ struct bt_conn_tx {
 	void *user_data;
 };
 
+enum acl_data_flags {
+	/** @brief Num Complete Sent from the Host */
+	ACL_DATA_HOST_NUM_COMPLETE_SENT = BIT(0),
+};
```

Comment:
Personal opinion that is optionally to consider: I do prefer when enums and their values share the same prefix, e.g.

```suggestion
enum acl_data_flags {
	/** @brief Num Complete Sent from the Host */
	ACL_DATA_FLAGS_HOST_NUM_COMPLETE_SENT = BIT(0),
};
```

Besides that, `enum`s are rarely in plural:
```suggestion
enum acl_data_flag {
	/** @brief Num Complete Sent from the Host */
	ACL_DATA_HOST_NUM_COMPLETE_SENT = BIT(0),
};
```